content/docs/operating/troubleshooting.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Troubleshooting
+sort_rank: 2
+---
+
+# Troubleshooting
+
+## Device Errors
+
+### Device is not showing up in Cluster/Talaria
+
+- Problem: Parados has the incorrect url.
+
+  - Solution: Validate Parados has the correct url.
```

Comment:
I think this is the first time I've read Parados anywhere? Should this say something like `mock CPE/device` instead? Or add Parados to other pages where they are talking about mock devices.